rnd/autogpt_server/migrations/20240910000943_add_analytics_to_backend/migration.sql

```diff
@@ -0,0 +1,53 @@
+-- CreateEnum
```

Comment:
Please can we have a single migration for the analytics additions

Reply:
There's not real additional cost to doing this but sure